scripts/linting/_check-etags

```diff
@@ -11,12 +11,9 @@ check_presence () {
 
 }
 
-check_presence analysis.js
 check_presence fetcher.js
 check_presence appsupport.js
 check_presence unsupportedBrowser.js
-check_presence analysis.js
```

Comment:
Why are these removed?

Reply:
That is, shouldn't there be an F# file to replace them with?

Reply:
`check_presence BlazorWorker.js` already exists within `_check-etags`